src/Maybe.js

```diff
@@ -95,4 +97,12 @@ _Nothing.prototype.getOrElse = function(a) {
   return a;
 };
 
+_Just.prototype.toString = function() {
+  return 'Maybe.Just(' + R.toString(this.value) + ')';
+};
+
+_Nothing.prototype.toString = function() {
+  return 'Maybe.Nothing()';
```

Comment:
are the parens necessary here? i'm treating `Nothing` a bit differently than in Sanctuary